MaterialSkin/HTML/material/html/js/browse-resp.js

```diff
@@ -1010,8 +1010,17 @@ function parseBrowseResp(data, parent, options, cacheKey, parentCommand, parentG
                         }
                     }
                 }
+                // sort field for role sorting might be null, if so make sure nulls stay below non-nulls even when reversed.
+                let loopNotNulls = [];
+                let loopNulls = [];
+                const firstNullPos = data.result.albums_loop.map(e => e.rolesort_name).indexOf('');
```

Comment:
Out of curiosity, what impact does this lookup have on loading large lists?

Reply:
It's bound to slow it down, I'll test with a 25,000 size albums list...

Reply:
Well then it depends on how much - as this is not something I'd ever really use. However, I really cannot see if having a significant impact. As stated, I was just curious.

Reply:
You're right, negligible: <1ms on a mid-power laptop. Rendering 25,000 items takes much longer. I suppose it's doing a binary chop on the array.